test.js

```diff
@@ -252,19 +266,19 @@ test('fileType.mimeTypes.has', t => {
 
 test('validate the input argument type', t => {
 	t.throws(() => {
-		fileType('x');
+		fileType.fromBuffer('x');
```

Comment:
This is now an async call, so this assertion is no longer correct. You need to use `t.throwsAsync`.

Reply:
Ah yes, missed that, thanks.